lib/fixed_length/parser.rb

```diff
@@ -0,0 +1,42 @@
+module FixedLength
+  class Parser
+    DEFAULT_TYPE = 'a'
+
+    attr_reader :columns, :body
+
+    def initialize(columns, body)
+      @columns = columns
+      @body = body
+    end
+
+    def parse
+      zip unpacked_body
+    end
+
+
+    private
+
+    def unpacked_body
+      body.lines.map do |line|
+        line.unpack(unpack_format)
```

Comment:
thanks :)

I need to benchmark this vs. more common ways to parse a fixed char length file.

I'm also thinking about releasing this `lib/fixed_width` as a simple gem, but it requires a file format in order for it to work. I'm not too sure how to approach that. 

Maybe package a `NullFormat` / `NoneFormat` class along with the gem so things work. Or just give it a default behavior if an instance of `Format` isn't found?